docs/dev/02-making-changes.md

```diff
@@ -49,7 +50,7 @@ Here are some tips on how to set up a Karma workspace and how to send a good pul
 
 - Build the client code via:
   ```bash
-  $ npm build
+  $ npm run build
```

Comment:
This is because recent NPM versions report error, when `npm build` is run:

```
$ npm build
npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?
$ npm run build

> karma@5.0.2 build /Users/devoto13/Projects/karma
> grunt build

Running "browserify:client" (browserify) task
>> Bundle static/context.js created.
>> Bundle static/karma.js created.

Done.
```